Don't strip debug symbols.
Backport hotfix patch commit b86390e4428bf38ce4c1ce0e9ca446d6d25955e8

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -63,7 +63,7 @@ detect_architecture("__arm__" ARM)
 detect_architecture("__aarch64__" ARM64)
 
 cmake_dependent_option(ENABLE_JIT "Enable JIT recompiler" ON
-    "(ARCHITECTURE STREQUAL x86_64 AND NOT APPLE) OR ARCHITECTURE STREQUAL ARM64" OFF)
+    "ARCHITECTURE STREQUAL x86_64 OR ARCHITECTURE STREQUAL ARM64;NOT ARCHITECTURE STREQUAL x86_64 OR NOT APPLE" OFF)
 cmake_dependent_option(ENABLE_JIT_PROFILING "Enable JIT profiling with VTune" OFF "ENABLE_JIT" OFF)
 option(ENABLE_OGLRENDERER "Enable OpenGL renderer" ON)
 
@@ -77,10 +77,6 @@ endif()
 
 if (ENABLE_LTO)
     set(CMAKE_INTERPROCEDURAL_OPTIMIZATION TRUE)
-endif()
-
-if (NOT APPLE)
-    set(CMAKE_EXE_LINKER_FLAGS_RELEASE "${CMAKE_EXE_LINKER_FLAGS_RELEASE} -s")
 endif()
 
 if (WIN32)
